packages/webdriverjs/test/test-utils.ts

```diff
@@ -1,21 +1,27 @@
 import { WebDriver, Builder } from 'selenium-webdriver';
-import chromedriver from 'chromedriver';
 import chrome from 'selenium-webdriver/chrome';
 import firefox from 'selenium-webdriver/firefox';
+import { config } from 'dotenv';
+import os from 'os';
+import path from 'path';
+
+const HOME_DIR = os.homedir();
+const BDM_CACHE_DIR = path.resolve(HOME_DIR, '.browser-driver-manager');
+
+config({ path: path.resolve(BDM_CACHE_DIR, '.env') });
 
 export const Webdriver = (): WebDriver => {
   // Weird type change since 4.23.1 release
   // @see https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/69724
   const options = new chrome.Options();
   options.addArguments('headless');
+  options.setBinaryPath(process.env.CHROME_TEST_PATH as string);
```

Comment:
What happens if the user doesn't run BDM before running these tests? I think right now it'll fall back to the globally installed versions, but I think it would be preferable to bail early with an actionable error message that explains the required BDM command; I think we'd prefer to force local devs to use the same setup we'll be testing against in CI.